packages/core/src/index.ts

```diff
@@ -431,10 +431,15 @@ class Hopr extends EventEmitter {
         }
 
         const channel = ethereum.getChannel(ticketIssuer, ticketReceiver)
-
         const channelState = await channel.usToThem()
 
-        if (channelState.status !== ChannelStatus.Open) {
+        if (typeof channelState === 'undefined') {
```

Comment:
Can we pre-set to `ChannelStatus.NonInitialized` instead of using `undefined`?

Reply:
this will require a small refactor, much rather not tackle it atm in this release

Reply:
LGTM then